runtime/rhala/src/lib.rs

```diff
@@ -780,6 +825,106 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
     type ReservedXcmpWeight = ReservedXcmpWeight;
 }
 
+parameter_types! {
+    pub const ClassDeposit: Balance = 100 * DOLLARS;
+    pub const InstanceDeposit: Balance = 1 * DOLLARS;
+    pub const KeyLimit: u32 = 32;
+    pub const ValueLimit: u32 = 256;
+    pub const StringLimit: u32 = 50;
+}
+
+impl pallet_uniques::Config for Runtime {
+    type Event = Event;
+    type ClassId = u32;
+    type InstanceId = u32;
+    type Currency = Balances;
+    type ForceOrigin = EnsureRoot<AccountId>;
+    type CreateOrigin = AsEnsureOriginWithArg<EnsureSigned<AccountId>>;
+    type Locker = ();
+    type ClassDeposit = ClassDeposit;
+    type InstanceDeposit = InstanceDeposit;
+    type MetadataDepositBase = MetadataDepositBase;
+    type AttributeDepositBase = MetadataDepositBase;
+    type DepositPerByte = MetadataDepositPerByte;
+    type StringLimit = StringLimit;
+    type KeyLimit = KeyLimit;
+    type ValueLimit = ValueLimit;
+    #[cfg(feature = "runtime-benchmarks")]
+    type Helper = ();
+    type WeightInfo = pallet_uniques::weights::SubstrateWeight<Runtime>;
+}
+
+parameter_types! {
+    pub const MaxRecursions: u32 = 10;
+    pub const ResourceSymbolLimit: u32 = 10;
+    pub const PartsLimit: u32 = 3;
+    pub const MaxPriorities: u32 = 3;
+    pub const CollectionSymbolLimit: u32 = 100;
+}
+
+impl pallet_rmrk_core::Config for Runtime {
+    type Event = Event;
+    type ProtocolOrigin = EnsureRoot<AccountId>;
+    type MaxRecursions = MaxRecursions;
+    type ResourceSymbolLimit = ResourceSymbolLimit;
+    type PartsLimit = PartsLimit;
+    type MaxPriorities = MaxPriorities;
+    type CollectionSymbolLimit = CollectionSymbolLimit;
+}
+
+parameter_types! {
+    pub const MaxPropertiesPerTheme: u32 = 100;
+    pub const MaxCollectionsEquippablePerPart: u32 = 100;
+}
+
+impl pallet_rmrk_equip::Config for Runtime {
+    type Event = Event;
+    type MaxPropertiesPerTheme = MaxPropertiesPerTheme;
+    type MaxCollectionsEquippablePerPart = MaxCollectionsEquippablePerPart;
+}
+
+parameter_types! {
+    pub const MinimumOfferAmount: Balance = DOLLARS / 10_000;
+}
+
+impl pallet_rmrk_market::Config for Runtime {
+    type Event = Event;
+    type ProtocolOrigin = EnsureRoot<AccountId>;
+    type Currency = Balances;
+    type MinimumOfferAmount = MinimumOfferAmount;
+}
+
+parameter_types! {
+    pub const SecondsPerEra: u64 = 86_400;
+    pub const MinBalanceToClaimSpirit: Balance = 10 * DOLLARS;
+    pub const LegendaryOriginOfShellPrice: Balance = 15_000 * DOLLARS;
+    pub const MagicOriginOfShellPrice: Balance = 10_000 * DOLLARS;
+    pub const PrimeOriginOfShellPrice: Balance = 500 * DOLLARS;
+    pub const IterLimit: u32 = 1_000;
+    pub const FoodPerEra: u32 = 5;
+    pub const MaxFoodFeedSelf: u8 = 2;
+    pub const IncubationDurationSec: u64 = 1_209_600;
+}
+impl pallet_pw_nft_sale::Config for Runtime {
+    type Event = Event;
+    type OverlordOrigin = EnsureRootOrHalfCouncil;
```

Comment:
Not blocking
Maybe we should call another name, e.g. `RootOrigin` or `GovernanceOrigin`
I was confused in `set_overlord`, why set overlord requires `OverlordOrigin`?

Reply:
I can change this to `GovernanceOrigin`